### PR TITLE
remove cloudservice UD

### PR DESCRIPTION
**Pull request by grizzlytheodore**

### CODE REVIEW

File: generator/processors/Microsoft.Compute.ts [hunk 1]
```diff
@@ -56,6 +56,14 @@ export const postProcessor: SchemaPostProcessor = async (namespace, apiVersion,
         schema.definitions.VirtualMachineScaleSetExtension.properties.properties = extensionsProperties(apiVersion);
     }
 
+    /// Remove CloudService Update Domain
+    if (schema.resourceDefinitinos?.cloudServices_updateDomains) {
```
Comment (anthony-c-martin) [inline generator/processors/Microsoft.Compute.ts:60]: > resourceDefinitinos

Typo!

**Comment by grizzlytheodore** [comment]
typo fixed. thank you for catching @anthony-c-martin

**Review by anthony-c-martin — APPROVED** [review]

**Comment by grizzlytheodore** [comment]
@anthony-c-martin please merge !

**Comment by anthony-c-martin** [comment]
> @anthony-c-martin please merge !

Oops, thought I had already!

**Pull request merged**